reaktive/src/jvmNativeCommonTest/kotlin/com/badoo/reaktive/utils/atomic/AtomicReferenceThreadingTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.badoo.reaktive.utils.atomic
+
+import com.badoo.reaktive.test.doInBackgroundBlocking
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertSame
+import kotlin.test.assertTrue
+
+class AtomicReferenceThreadingTest {
```

Comment:
Add test for freeze too? Verify that in some cases referenced value is frozen and in other is not.